clients/client-alexa-for-business/index.ts

```diff
@@ -113,3 +113,4 @@ export * from "./commands/UpdateRoomCommand";
 export * from "./commands/UpdateSkillGroupCommand";
 export * from "./pagination/Interfaces";
 export * from "./models/index";
+export * from "./models/index";
```

Comment:
This has been fixed in https://github.com/awslabs/smithy-typescript/pull/322, can you pull in the latest Smithy-typescript and regenerate them?